javascript/index.js

```diff
@@ -14,10 +14,20 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
+  secUni.innerHTML = chronometer.getSeconds()
```

Comment:
good approach, this helps you check that the DOM is being updated every second! However the idea of this function is that it is passed as a callback, and it will call the other functions that will update the DOM (printMinutes and printSeconds)